css/teams/5/blog-posts.css

```diff
@@ -0,0 +1,655 @@
+hr {
+  border-top: solid 1px var(--color-gray-600);
+}
+
+.hero-header {
+  display : flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--color-primary-800);
+  padding-top: 6em;
+}
+
+.ut-top-navbar {
+  border: none;
+}
+
+.ut-hero-header-text-position {
+  margin-top: 50px;
+  display : flex;
+  flex-direction: column;
+  align-items: center;
+}
+.ut-hero-header-text-position span {
+  color: var(--color-primary-200);
+}
+.ut-text-md-med {
+  display: block;
+  color:  var(--color-primary-200);
+}
+
+.hero-header .ut-display-2xl-semi-bold {
+ color: #FFFFFF; 
+ text-align: center;
+}
+
+.hero-header .ut-text-xl-reg {
+  color: var(--color-primary-200);
+}
+
+.ut-button-container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: space-around;
+  width: 306px;
+  height: 60px;
+  margin-top: 40px;
+}
+
+.button {
```

Comment:
can we please add another class name that describes better the action that is intended with this button.